### DISCUSSION TITLE
how to uninstall

### DISCUSSION THREAD

**Discussion by derFleder** [QL-Win/QuickLook#1128] [Q&A]
Hello,

I wanted to uninstall this, but I found not a single clue as to how to manage this.
It does not show up in my programs on windows.
Please give me a hint, thank you.

best regards

**Comment by rabelux** [comment]
Does #109 answer your question?

**Comment by derFleder** [comment]
@rabelux sorry for the late reply and thank you for your time.

Unfortunately, neither of those two methods worked.
It does not appear in under programs/apps and in my appdata folder is no trace of the mentioned files/folders.

When I check it via Task-Manager and open the patch to that .exe, it tells me its located in \Programms\WindowsApps\
So I thought it might be installed via the Microsoft Store? It is listed under "installed" but there is no way or button to uninstall it there.

I am completely clueless here.

**Comment by xupefei** [comment]
Yes, it's installed via Microsoft Store. There's a "uninstall" button when right-clicking on the application icon in your start menu.

**Comment by techsrikanth369** [comment]
This method works for those who downloaded from Microsoft store, 

If you're having trouble removing a QuickLook plugin and the standard methods aren't working, here's what worked for me.

Locate the **QuickLook.Plugin** folder  

I used a third-party search tool named **"Everything"** to scan my entire system for the QuickLook.Plugin folder.

Once I found it, I navigated to the folder, deleted the specific plugin folder I wanted to remove, and restarted QuickLook.

After the restart, the plugin was successfully removed.

If you're facing a similar issue, I recommend searching your system for the QuickLook.Plugin folder—this approach worked perfectly for me!
